.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "libs/EVT-core"]
+	path = libs/EVT-core
+	url = ../EVT-core.git
+    # Need to set the full url if not cloned into the EVT organization
```

Comment:
This relative path is really nice, but it only works if the template is cloned into the EVT organization.  And only @cbolles has permissions to do that.  I think it's fine, but I want to draw attention to this.

Reply:
readthedocs.org actual fails if you set this as the absolute path, so it either needs to be HTTP or relative.